modules/ch-subdivisions.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * The list of subdivisions of Uruguay based on ISO 3166-2:CH standard.
+ *
+ * Source: https://en.wikipedia.org/wiki/ISO_3166-2:CH ISO 3166-2:CH
+ */
+class Listo_CH_Subdivisions implements Listo {
+	private function __construct() {}
+
+	public static function items() {
+		return array(
+			'AG' => _x( 'Aargau','ch_subdivisions','listo' ),
+			'AI' => _x( 'Appenzell Innerrhoden','ch_subdivisions','listo' ),
+			'AR' => _x( 'Appenzell Ausserrhoden','ch_subdivisions','listo' ),
+			'BE' => _x( 'Bern','ch_subdivisions','listo' ),
+			'BL' => _x( 'Basel-Land','ch_subdivisions','listo' ),
```

Comment:
Use the name that is used in the source data ([https://en.wikipedia.org/wiki/ISO_3166-2:CH](https://en.wikipedia.org/wiki/ISO_3166-2:CH))

Reply:
Sorry, should now be perfect fitted and sorted.